feedfinder/feedfinder.go

```diff
@@ -3,52 +3,68 @@ package feedfinder
 import (
 	"context"
 	"net/http"
+	"net/url"
 
 	"github.com/PuerkitoBio/goquery"
 	"github.com/sapuri/feed-finder/feedfinder/errors"
 )
 
-type FeedFinder struct{}
+type Feed struct {
+	Title string
+	URL   *url.URL
+}
```

Comment:
[note] I think it's worth exposing the `FeedFinder` interface and `Feed` (=result) to the users